Add bridged docker network Aeria

mrmarkuz opened the pull request:
I added two actions and the db prop bridge for the docker service:

* nethserver-docker-create-aeria - installs the plugin and creates the network when db prop bridge exists
* nethserver-docker-upgrade-plugin - upgrades the plugin

Feel free to add an event, I can do it tomorrow evening...

mrmarkuz committed:
Add bridged docker network Aeria

stephdl commented on root/etc/e-smith/events/actions/nethserver-docker-create-aeria:
```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+#
+# Copyright (C) 2018 Nethesis S.r.l.
```
adjust the date

stephdl commented on root/etc/e-smith/events/actions/nethserver-docker-upgrade-plugin:
```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+#
+# Copyright (C) 2018 Nethesis S.r.l.
```
adjust the date

stephdl commented:
I have some ideas , but probably really cosmetic

we need to use a createlinks to run these two actions
nethserver-docker-create-aeria must be used  by the event nethserver-docker-update
nethserver-docker-upgrade-plugin must be used by the event nethserver-docker-plugin-update ?????

we need an example how run a container in aeria in the README of this repo

I have some doubt on the name of the prop, `bridge`, do you think we could have several other network on several other bridge, it could make sense  in that case to name the prop `bridgeAeria`

good work....once the createlinks and the readme updated I cannot wait to test it :)

mrmarkuz committed:
Add aeria events and rename bridge prop

mrmarkuz commented:
Events added, fixed the dates and added some aeria documentation to README

stephdl commented:
I cannot create a container on `aeria` with the command you given in the readme, can you reproduce ?
```
[root@ns7loc10 ~]# docker network ls
NETWORK ID          NAME                DRIVER                       SCOPE
387068b4ec11        aeria               devplayer0/net-dhcp:latest   local
13b28120976f        aqua                bridge                       local
6de63e37c59a        host                host                         local
38b83e918991        none                null                         local
[root@ns7loc10 ~]#   docker run -d --name pihole --net=aeria --mac-address=01:02:03:04:05:06 pihole/pihole:latest
Unable to find image 'pihole/pihole:latest' locally
latest: Pulling from pihole/pihole
6d28e14ab8c8: Pull complete 
288c0d516484: Pull complete 
aa113f1706df: Pull complete 
abb9c2faee3c: Pull complete 
f84b063eb6ef: Pull complete 
b4630b351376: Pull complete 
371ee929429b: Pull complete 
e2ec63a5def2: Pull complete 
575de6c8051a: Pull complete 
bfea53210c88: Pull complete 
612b8aa9e362: Pull complete 
8970f2277548: Pull complete 
96555ea1a5d5: Pull complete 
Digest: sha256:d586f1a05c2a81b224d2c30d33ab9081b8baa0471f87d900d55e6155ed89cb08
Status: Downloaded newer image for pihole/pihole:latest
1df72097f391a9be224b6284b8d642b75e7738e9b1e53ebf789251256dadc5c7
docker: Error response from daemon: failed to create endpoint pihole on network aeria: NetworkDriver.CreateEndpoint: local variable 'res' referenced before assignment.
```

stephdl commented:
I pushed a commit directly to master for createlinks, hence the conflict, ping me if you have some troubles to solve it

mrmarkuz commented:
My fault. I simplified the example in README to not confuse people with a lot of parameters but it seems I simplified too much. I am going to find a simpler container for the README example.
The full line for pihole to test:
`docker run -d --name pihole -e TZ="Europe/Vienna" -e WEBPASSWORD="admin" -v "$(pwd)/etc-pihole/:/etc/pihole/" -v "$(pwd)/etc-dnsmasq.d/:/etc/dnsmasq.d/" --cap-add NET_ADMIN --net=aeria --mac-address=0e:6f:47:f7:26:1a --restart=unless-stopped pihole/pihole:latest`

stephdl reviewed (approved):
workable

mrmarkuz committed:
Merge branch 'master' into master
